CarRental/CarRental.Web/Views/Car/_AvailableOptionsForAddons.cshtml

```diff
@@ -1,6 +1,4 @@
-﻿@ViewData.Eval("Addons")
```

Comment:
czemu to w sumie wywaliles

Reply:
to ten wpis okazał się bardzo upierdliwy, bo powodował wyświetlanie w widoku napisu w rodzaju "System......" (czyli jakby obiektu  do stringa);
dało się inaczej, tak jak teraz